### ISSUE TITLE
WFdownloads 3.3 - Files download

### ISSUE THREAD

**Issue by Aminov81** [XoopsModules25x/wfdownloads#29]
The download of files is impossible. The page "visit.php" is inaccessible (ERR_INVALID_RESPONSE).
This issue concerns only the files. When there is just an external link the redirection is done well.

**Comment by mambax7** [comment]
Is this your local file? 
I tested it with the Publisher (in the testdata) that pointed to the GitHub file, and it worked without problems.

**Comment by Aminov81** [comment]
Yeah the problem concerns only local files (hosted in my server).
No problem with external links.

**Comment by mambax7** [comment]
Can you use a full local link, e.g.

http://mysite.com/uploads/files/myfile.zip

and see if it works

**Comment by Aminov81** [comment]
I tested with full link but it didn't work unfortunately..

**Comment by mambax7** [comment]
Where do you have the local files? The should be in the folder:
/uploads/wfdownloads/

**Comment by alain01** [comment]
Any problem with the https site and the internal link is http ? (a no secure link in a secure site)
Just my tiny Idea...

**Comment by Aminov81** [comment]
The files are located here : /home/mawaridm//public_html/uploads
And that's the path i indicated in the prefrences of the module.
I thought i could choose any folder with the condition to indicate it in the preferences of the module.
I'll try now with this : /home/mawaridm//public_html//uploads/wfdownloads/

**Comment by Aminov81** [comment]
@alain01 
No, the https site is working fine.

**Comment by Aminov81** [comment]
I set the downloads path in the preferences to this : /home/user/public_html/uploads/wfdownloads
I created a new download and i uploaded it directly from the "downloads.php" page. The file is well uploaded and I see it in the folder /home/user/public_html/uploads/wfdownloads.
But instead of that the downloading is still not working !!

**Comment by alain01** [comment]
No, i think you melt root apache directory and home XOOPS directory.
/home/user/public_html/ is the root doc for Apache
for XOOPS, the upload directory is in "/uploads/wfdownloads", not "/home/user/public_html/uploads/wfdownloads".
It's just my idea ;-)
Sorry, i dont use wfdownload.

**Comment by Aminov81** [comment]
That's what i am using in my running website (wfdownloads 3.2) and it's working fine.
On the other hand, there is no problem with uploading. Wfdownloads is uploading files in the right directory. There is problem only with downloading.
Ok i'll do test with "/uploads/wfdownloads".

**Comment by Aminov81** [comment]
When i use this path "/uploads/wfdownloads", wfdownloads says "warning:the downloads folder doesn't exist" !
